modules/rename.sh

```diff
@@ -1,11 +1,20 @@
 #!/bin/bash
 
 function rename () {
-  CURRENT_HOSTNAME=$(< /etc/hostname tr -d " \\t\\n\\r")
-  echo "$1" > /etc/hostname
-  sed -i "s/127.0.1.1.*$CURRENT_HOSTNAME/127.0.1.1\\t$1/g" /etc/hosts
-  hostname "$1"
-  echo "Success: the hostname has been modified"
+  if
+    [ "$(expr "$1" : ".*[!@#\$%^\&*()_+].*")" -gt 0 ] || #checks for special characters
```

Comment:
@irisb1701 think we need to check if $1 is composed of a-z A-z 0-9 and - cause in ascii there are more then this special characters

Reply:
Ok just made the changes